joss_paper/paper.md

```diff
@@ -0,0 +1,121 @@
+---
+title: 'Zarrtraj: A Python package for streaming molecular dynamics trajectories from cloud services'
+tags:
+  - streaming
+  - molecular-dynamics
+  - file-format
+  - mdanalysis
+  - zarr
+authors:
+  - name: Lawson Woods
+    orcid: 0009-0003-0713-4167
+    affiliation: 1 
+  - name: Hugo Macdermott-Opeskin
+    orcid: 0000-0002-7393-7457
+    affiliation: 1
+  - name: Edis Jakupovic 
+    affiliation: 1
+  - name: Yuxuan Zhuang
+    orcid: 0000-0003-4390-8556
+    affiliations: 1
+  - name: Richard J Gowers
+    orcid: 0000-0002-3241-1846
+    affiliations: 1
+  - name: Oliver Beckstein
+    orcid: 000-0003-1340-0831
+    affiliation: 1
```

Comment:
For myself [1,2], Edis [1, 2]:

```
 - name: Department of Physics, Arizona State University, Tempe, Arizona, United States of America
   index: 1
 - name: Center for Biological Physics, Arizona State University, Tempe, AZ, United States of America
   index: 2

 ```
For yourself: something similar but for engineering and you could also add a present affiliation for SMS.